tests/fast-integration/skr-svcat-migration-test/skr-svcat-migration-test.js

```diff
@@ -85,6 +85,10 @@ describe("SKR SVCAT migration test", function() {
     secretsAndPresets = await sampleResources.storeSecretsAndPresets()
   });
 
+  it(`Should check if pod presets injected secrets to functions containers`, async function() {
+    await t.checkPodPresetEnvInjected();
```

Comment:
this goes over the pods that are created by the kyma `Functions` created by this test (`svcat-auditlog-api-1`, `svcat-auditlog-management-1` and `svcat-html5-apps-repo-1`) contain a pod that runs the function. It expects exactly 1 running pod, execs into it using kubectl and tests for following env variables:
- `uaa`, `url`, `vendor` for functions `svcat-auditlog-api-1`, `svcat-auditlog-management-1`
- `grant_type`, `saasregistryenabled`, `sap.cloud.service`, `uaa`, `uri`, `vendor` for function `svcat-html5-apps-repo-1`